components/payment/AccountTabContent.tsx

```diff
@@ -16,7 +16,7 @@ const IMAGE_SIZE = 100;
 
 const AccountTabContent: React.FC<AccountTabProps> = () => {
   return (
-    <View style={styles.contentContainer}>
+    <>
```

Comment:
this too

Reply:
In this file I add because it has multiple components. (Views and Button)

Reply:
I see then it's fine